backend/database.py

```diff
@@ -123,6 +123,22 @@ def getUserByName(conn, double_name):
     except Error as e:
         logger.debug(e)
 
+def get_user_by_device_id(conn, device_id):
+    # TODO @single_core: is this safe?
```

Comment:
No it is not, please look at "database.py#select_from_userapps" on how to properly insert.